aten/src/ATen/native/native_functions.yaml

```diff
@@ -1136,6 +1136,11 @@
 
 - func: max_pool2d(Tensor self, int[2] kernel_size, int[2] stride=[], int[2] padding=0, int[2] dilation=1, bool ceil_mode=False) -> Tensor
 
+- func: mkldnn_max_pool2d(Tensor self, int[2] kernel_size, int[2] stride=[], int[2] padding=0, int[2] dilation=1, bool ceil_mode=False) -> Tensor
+  requires_tensor: True
```

Comment:
requires_tensor is probably not required here

Reply:
yep just for documentation purpose